repository/src/main/java/com/outdoor/foodcalc/domain/repository/product/ProductRefRepo.java

```diff
@@ -101,6 +104,13 @@ public boolean deleteDishProducts(long dishId) {
         return jdbcTemplate.update(DELETE_DISH_PRODUCTS_SQL, parameters) > 0;
```

Comment:
lets change this method to return number of deleted records and remove `dishProductsExist`